internal/providers/terraform/aws/kms_key.go

```diff
@@ -0,0 +1,42 @@
+package aws
+
+import (
+	"github.com/infracost/infracost/internal/schema"
+	"github.com/shopspring/decimal"
+)
+
+func GetNewKMSKeyRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "aws_kms_key",
+		RFunc: NewKMSKey,
+	}
+}
+
+func NewKMSKey(d *schema.ResourceData, u *schema.ResourceData) *schema.Resource {
+
+	region := d.Get("region").String()
+	cmkCount := int64(1)
+
+	costComponents := []*schema.CostComponent{
+		{
+			Name:            "CMK",
+			Unit:            "months",
+			UnitMultiplier:  1,
+			MonthlyQuantity: decimalPtr(decimal.NewFromInt(cmkCount)),
+			ProductFilter: &schema.ProductFilter{
+				VendorName:    strPtr("aws"),
+				Region:        strPtr(region),
+				Service:       strPtr("awskms"),
+				ProductFamily: strPtr("Encryption Key"),
+				AttributeFilters: []*schema.AttributeFilter{
+					{Key: "usagetype", ValueRegex: strPtr("/-KMS-Keys/")},
```

Comment:
```suggestion
					{Key: "locationType", Value: strPtr("AWS Region")},
					{Key: "usagetype", ValueRegex: strPtr("/KMS-Keys/")},
```
This ensures that if/when AWS Outpost support is added for this resource, the filters still return the price we want.